samples/Demo/speedometer-consumer/src/main.rs

```diff
@@ -174,6 +210,14 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     #[cfg(target_arch = "aarch64")]
     {
     let mut panel = led_driver::init();
+    const default_led_count: i32 = 8;
+    const default_led_delay_ms: u64 = 1000;
+    let led_count = getCmdArg(LED_COUNT_FLAG.to_string(), default_led_count);
+    let led_delay_ms = getCmdArg(LED_DELAY_MS_FLAG.to_string(), default_led_delay_ms);
+    let led_color = getCmdArg(LED_COLOR_FLAG.to_string(), RED_RGB_COLOR);
+
+    runningLed(&mut panel, led_color, led_delay_ms, led_count);
+
     led_driver::setAllLedsToRgb(&mut panel, 0x00200000);
```

Comment:
lets remove most of this, please leave:
1. set all leds to some color (can be red, does not matter)
2. delay 1000ms
3. show some gradient
4. delay 1000ms
5. show gradient with modified start/stop
6. delay 1000ms
7. go into the running led code

Reply:
Modified